x/ccv/provider/keeper/key_assignment_test.go

```diff
@@ -47,7 +47,7 @@ func TestGetAllValidatorConsumerPubKey(t *testing.T) {
 	pk, ctx, ctrl, _ := testkeeper.GetProviderKeeperAndCtx(t, testkeeper.NewInMemKeeperParams(t))
 	defer ctrl.Finish()
 
-	rand.Seed(time.Now().Unix()) // nolint:staticcheck // ignore SA1019 for tests
+	rand.Seed(time.Now().Unix())
```

Comment:
This diff is neccessary because the linter has different functionality when using go 1.19 vs 1.20